http/1.6/modules/ROOT/pages/http-request-ref.adoc

```diff
@@ -39,41 +95,107 @@ By default, Mule defines different limits for the HTTP request size, response si
 | Response Size | `org.glassfish.grizzly.nio.transport.TCPNIOTransport.max-send-buffer-size` | `1048576` (1MB)
 |===
 
-=== Headers
+To learn how to set system properties in the `wrapper.conf` file, refer to the xref:mule-runtime::mule-app-properties-system.adoc#set-properties[Set System Properties in the wrapper.conf File] documentation.
+
+== Configure Headers
+
+To add headers to an HTTP request, you can configure two options:
 
-Select *Headers* in the *General* configuration to add headers to the request. For example, add header names `HeaderName1` and `HeaderName2`, add header values `HeaderValue1` and `HeaderValue2`. You can use DataWeave expressions, for example `#[{'HeaderName1' : 'HeaderValue1', 'HeaderName2' : 'HeaderValue2'}]`.
+* *Headers* +
+At the HTTP *Request* operation level
+* *Default headers* +
+At the HTTP *Request* global configuration level
 
-There are also use cases in which there are a set of default headers you always want to set. For example, you might want to always send authentication headers like `x-csrf-token: Fetch`.
+You configure default headers at the HTTP global configuration level so that you don't have to specify every header on every single request. With this configuration, those headers are added to every outbound request, alongside any headers you configure at the HTTP *Request* operation level.
```

Comment:
```suggestion
Configure default headers at the HTTP global configuration level to avoid specifying every header on every single request. With this configuration, those headers are added to every outbound request, alongside any headers you configure at the HTTP *Request* operation level.
```